ffcx/codegeneration/definitions.py

```diff
@@ -79,8 +131,8 @@ def coefficient(self, t, mt, tabledata, quadrature_rule, access):
         assert begin < end
 
         # Get access to element table
-        FE = self.symbols.element_table(tabledata, self.entitytype, mt.restriction)
-        ic = self.symbols.coefficient_dof_sum_index
+        # FE = self.symbols.element_table(tabledata, self.entitytype, mt.restriction)
+        FE = self.symbols.table_access(tabledata, self.entitytype, mt.restriction, iq, ic)
```

Comment:
Probably should be in `access.py` not in `symbols.py` ?

Reply:
Access is not part of `FFCXBackendDefinitions`.
I think it would require some restructuring.